src/Importer.tsx

```diff
@@ -35,33 +35,40 @@ function Importer({ show, onImport, onClose }: ImportProps): JSX.Element {
   async function parseFile(file: File) {
     setLoading(true);
 
+    let blobs: Array<Blob> = [];
+    let games: Array<Game> = [];
+
     let blob: Blob = file;
     if (file.name.endsWith(".zip")) {
       const reader = new zip.ZipReader(new zip.BlobReader(blob));
       const entries = await reader.getEntries();
-      if (entries.length !== 1) {
-        return setError("Zip files should contain only one log file.");
+      for (let entry of entries) {
+        blobs.push(await entry.getData!(new zip.BlobWriter()));
```

Comment:
I've generally been trying to avoid loading the whole raw blobs into memory, so would prefer to avoid this if possible

Not that I'm entirely clear how much of a `Blob` is loaded into the JS memory by `entry.getDatta`

Reply:
Right...

So the correct thing to do, I assume, would be to extract the parsing into a function and run it for each entry.

Reply:
Yeah, or possibly create the parser earlier, then have a function which accepts blobs and feeds them into the parser instance it closes over

Then that func can be called in the zip branch's loop and the non-zip branch